packages/@react-types/searchfield/package.json

```diff
@@ -9,7 +9,9 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "dependencies": {
-    "@react-types/textfield": "^3.2.3"
+    "@react-types/combobox": "^3.0.1",
+    "@react-types/shared": "^3.7.1",
+    "@react-types/textfield": "^3.2.1"
```

Comment:
not sure this should be less than it used to be, what caused this change?

Reply:
Handled by creating autocomplete package via #2371.